### PR TITLE
Updates Guidelines for suspension or removal

### PR DESCRIPTION
**Pull request by plehegar**
These changes allow:

1. Chair/TC is always expected to send a warning to the person before further actions
2. Allow Chair/TC to suspend a participant temporarily, instead of waiting for the CEO to step in
3. Make it clear that any disagreement between Chairs or TCs can be resolved by going to the CEO

This is a follow-up to https://github.com/w3c/w3process/issues/592

[Diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FGuide%2Fprocess%2Fbanning.html&doc2=https%3A%2F%2Fraw.githubusercontent.com%2Fw3c%2FGuide%2Fsuspension-2022%2Fprocess%2Fbanning.html)

### CODE REVIEW

File: process/banning.html [hunk 1]
```diff
@@ -44,37 +44,55 @@ <h4>Nearby</h4>
...
-
-<p>2. Following violation of the <a href="https://www.w3.org/Consortium/cepc/">W3C Code of Ethics and Professional Conduct</a>, at least one warning by the chair(s), and at least one subsequent violation of the CEPC, chairs may temporarily or permanently suspend participation by a person who has not formally joined the group.</p>
-
-<p>3. A W3C group chair and team contact may propose to suspend or remove a participant from a single group - typically for egregious and repetitive CEPC violations - by making a request to the CEO.</p>
+</li>
+<li><p>In case of disagreement between the W3C group chair and team contact, each should explain their reasons to the CEO.</p></li>
+<li>
+<p>Following violation of the <a href="https://www.w3.org/Consortium/cepc/">W3C Code
+of Ethics and Professional Conduct</a>, a W3C group chair and team contact should
+give at least one written warning. If the person is a is a W3C Member representative
```
Comment (swickr) [inline process/banning.html:66]: s/is a is a/is a/

File: process/banning.html [hunk 2]
```diff
@@ -44,37 +44,55 @@ <h4>Nearby</h4>
...
 <p>If the chair, team contact, CEO, COO is the one accused of unacceptable behavior, they should not be involved in the decision to ban.</p>
-
-<p>2. Following violation of the <a href="https://www.w3.org/Consortium/cepc/">W3C Code of Ethics and Professional Conduct</a>, at least one warning by the chair(s), and at least one subsequent violation of the CEPC, chairs may temporarily or permanently suspend participation by a person who has not formally joined the group.</p>
-
-<p>3. A W3C group chair and team contact may propose to suspend or remove a participant from a single group - typically for egregious and repetitive CEPC violations - by making a request to the CEO.</p>
+</li>
+<li><p>In case of disagreement between the W3C group chair and team contact, each should explain their reasons to the CEO.</p></li>
+<li>
+<p>Following violation of the <a href="https://www.w3.org/Consortium/cepc/">W3C Code
+of Ethics and Professional Conduct</a>, a W3C group chair and team contact should
```
Comment (swickr) [inline process/banning.html:65]: The change from the current text to "... should give ..." is a subtle one.  This is advising (both) the chair and team contact to give a warning.  The original text was specific to suspension and said that suspension is only allowed after a warning, not that a warning must be issued.

Is this change intentional?

File: process/banning.html [hunk 3]
```diff
@@ -44,37 +44,55 @@ <h4>Nearby</h4>
...
+give at least one written warning. If the person is a is a W3C Member representative
+participant in the Group, the Member's Advisory Committee representative must be
```
Comment (swickr) [inline process/banning.html:67]: "Member representative participant" reads awkwardly.  I suggest "If the participant was appointed by a Member, the Member's Advisory ..."

File: process/banning.html [hunk 4]
```diff
@@ -44,37 +44,55 @@ <h4>Nearby</h4>
...
       <ul>
-        <li>If the group chair and team contact are not in agreement, each should explain their reasons.</li>
         <li>If the participant is a W3C Member representative, the CEO will coordinate the response with the Member's Advisory Committee representative.</li>
         <li>If anyone raises an issue to the CEO that a participant fails to meet the requirements of <a href="https://www.w3.org/Consortium/Process/#ParticipationCriteria">Individual Participation Criteria</a>, then the CEO is empowered to investigate. If the participant indeed is failing to meet those requirements, then after at least one warning and at least one subsequent violation, the CEO may temporarily or permanently suspend participation.</li>
+        <li>To avoid potential rapid escalation of unacceptable behavior, a W3C group chair and team contact may temporarily suspend a participant from a single group for up to 5 business days after alerting the CEO (and the Member's Advisory Committee representative if the participant is not an Invited Expert). Any subsequent suspension within 6 months would need to be requested from the CEO.</li>
```
Comment (swickr) [inline process/banning.html:81]: Consider emphasizing that the chair and team contact must act together: "... a W3C group chair together with a team contact may ..."

File: process/banning.html (hunk 2 above)
Comment (plehegar) [reply process/banning.html]: the change was not intentional. I refactored the proposal to avoid the mistake.

**Comment by plehegar** [comment]
Note that the part "If anyone raises an issue to the CEO that a participant fails to meet the requirements" was not intended to be subjected to the Chairs asking for a removal or suspension. So I moved it up out of that sub-bullet.

**Comment by plehegar** [comment]
Thanks to @swickr and @jeffjaffe for their initial feedback. This is ready for wider review.

File: process/banning.html [hunk 5]
```diff
@@ -44,37 +44,48 @@ <h4>Nearby</h4>
...
+</li>
+<li><p>In case of disagreement between the W3C group chair and team contact, each should explain their reasons to the CEO.</p></li>
+<li>
+  <p>Following violation of the <a href="https://www.w3.org/Consortium/cepc/">W3C Code of Ethics and Professional Conduct</a>, at least one warning by the chair(s), and at least one subsequent violation of the CEPC, a W3C group chair and team contact, acting together, may:</p>
+  <ol>
+<li>
+  <p>temporarily or permanently suspend participation by a person if the person has not formally joined the group.</p>
+</li>
+<li>
+  <p>temporarily suspend participation for up to 5 business days if the person is a participant the group after emailing the CEO (and the Member's Advisory Committee representative if the participant is not an Invited Expert).  Any subsequent suspension within 6 months would need to be requested from the CEO.</p>
```
Comment (swickr) [inline process/banning.html:70]: "... is a participant _in_ the group..."

File: process/banning.html [hunk 6]
```diff
@@ -44,37 +44,48 @@ <h4>Nearby</h4>
...
+</li>
+<li>propose to suspend or remove a participant from a single group - typically for egregious and repetitive CEPC violations - by making a request to the CEO. If the participant is a W3C Member representative, the CEO will coordinate the response with the Member's Advisory Committee representative.</li>
+</li>
+</ol>
+</li>
+<li>If anyone raises an issue to the CEO that a participant fails to meet the requirements of <a href="https://www.w3.org/Consortium/Process/#ParticipationCriteria">Individual Participation Criteria</a>, then the CEO is empowered to investigate. If the participant indeed is failing to meet those requirements, then after at least one warning and at least one subsequent violation, the CEO may temporarily or permanently suspend participation.</li>
```
Comment (swickr) [inline process/banning.html:76]: suggest enclosing this item in <p> to match the formatting of the other items.

**Comment by nigelmegitt** [comment]
Diff link is out of date: [diff for most recent commit at time of posting this comment](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FGuide%2Fprocess%2Fbanning.html&doc2=https%3A%2F%2Fraw.githubusercontent.com%2Fw3c%2FGuide%2F8c5f9614e2db3e60a4ac7d0a470f3f31e7793210%2Fprocess%2Fbanning.html)

File: process/banning.html (hunk 6 above)
Comment (nigelmegitt) [inline process/banning.html:76]: This "raises an issue to the CEO" seems to run against the CEPC's section on [Reporting](https://www.w3.org/Consortium/cepc/#Reporting), which suggests getting assistance from an Ombudsperson or senior W3C Management.

File: process/banning.html [hunk 7]
```diff
@@ -44,37 +44,48 @@ <h4>Nearby</h4>
...
-<p>6. No individual may be banned from sending email to the W3C Ombuds team. (Bots may be banned.)</p>
-
-<p>7. The W3C Systems team may block spammers and others who abuse our systems.</p>
-
-<p>8. The W3C Systems team will provide technical support to implement suspensions.</p>
-
+</li>
+<li><p>In case of disagreement between the W3C group chair and team contact, each should explain their reasons to the CEO.</p></li>
+<li>
+  <p>Following violation of the <a href="https://www.w3.org/Consortium/cepc/">W3C Code of Ethics and Professional Conduct</a>, at least one warning by the chair(s), and at least one subsequent violation of the CEPC, a W3C group chair and team contact, acting together, may:</p>
```
Comment (nigelmegitt) [inline process/banning.html:64]: The CEPC's [Reporting](https://www.w3.org/Consortium/cepc/#Reporting) section appears to grant greater rights than this, saying:
> take such immediate action as they deem necessary 

Since the CEPC is mandated by the Process, I think we need to bring these guidelines into alignment with what the CEPC says.

**Review by nigelmegitt — CHANGES_REQUESTED** [review]
Seems to me that the changes made here, while an improvement to current guidance, don't actually match what the CEPC says, and I believe that takes priority, so possibly much greater rework than this is needed.

File: process/banning.html [hunk 8]
```diff
@@ -44,37 +44,48 @@ <h4>Nearby</h4>
...
+  <ol>
+<li>
+  <p>temporarily or permanently suspend participation by a person if the person has not formally joined the group.</p>
+</li>
+<li>
+  <p>temporarily suspend participation for up to 5 business days if the person is a participant the group after emailing the CEO (and the Member's Advisory Committee representative if the participant is not an Invited Expert).  Any subsequent suspension within 6 months would need to be requested from the CEO.</p>
+</li>
+<li>propose to suspend or remove a participant from a single group - typically for egregious and repetitive CEPC violations - by making a request to the CEO. If the participant is a W3C Member representative, the CEO will coordinate the response with the Member's Advisory Committee representative.</li>
```
[the remark this answers is not in the document]
Comment (koalie) [reply process/banning.html:72]: fwiw, +1 to remove "egregious or repetitive".

**Comment by koalie** [comment]
I suggest to remove "within 6 months" because it complicates things and is harder to check/verify and enforce.

**Comment by nigelmegitt** [comment]
> I suggest to remove "within 6 months" because it complicates things and is harder to check/verify and enforce.

That was added because previously there was no time limit on it at all, meaning that after one escalation no Chair or team member could _ever_ take further action without going via the CEO. That seemed unreasonable to me, and I felt that some duration should be specified after which reversion to the normal state for any participant would occur.

**Comment by plehegar** [comment]
Note: Need to clarify who is expected to communicate a decision.

**Comment by brewerj** [comment]
Please note request here https://github.com/w3c/w3process/issues/592#issuecomment-1130490019

**Comment by plehegar** [comment]
> Diff link is out of date: [diff for most recent commit at time of posting this comment](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FGuide%2Fprocess%2Fbanning.html&doc2=https%3A%2F%2Fraw.githubusercontent.com%2Fw3c%2FGuide%2F8c5f9614e2db3e60a4ac7d0a470f3f31e7793210%2Fprocess%2Fbanning.html)

I fixed the link in the original comment. It's now tied to the branch itself rather than a commit, so should always be up-to-date.

**Comment by plehegar** [comment]
(withdrawing this pull request for now)
